src/api.ts

```diff
@@ -70,6 +70,10 @@ router.get(`/:type(${TYPE_CONSTRAINTS})/:id`, async (req, res) => {
     return res.status(500).json({ status: 'error', error: 'failed to load content' });
   }
 
+  if (resolver && !resolverExists(type, resolver)) {
```

Comment:
What if we just use resolve as usual, and just throw an error if it does not exist, instead of using a condition ?

Reply:
Validation should be done on the api.ts side, we should consider that everything passed to the resolver to be valid, hence throwing an error would make sense.

I agree there's currently no proper validation on the resolver query params, which should be fine, as it's a debug params, used only for internal purpose.